lib/publiccloud/instance.pm

```diff
@@ -250,6 +251,7 @@ sub wait_for_ssh
     }
 
     # Debug output: We have ocasional error in 'journalctl -b' - see poo#96464 - this will be removed soon.
+    script_run("ssh -v $args{username}\@$args{public_ip} true", timeout => 360);
```

Comment:
Why?

Reply:
Yes, but we see why it will fail. There is `-v`.

Reply:
I see, OK